api/v1beta1/hosted_controlplane.go

```diff
@@ -211,16 +219,15 @@ type HostedControlPlaneStatus struct {
 	// +kubebuilder:validation:Optional
 	OAuthCallbackURLTemplate string `json:"oauthCallbackURLTemplate,omitempty"`
 
-	// Version is the semantic version of the release applied by
-	// the hosted control plane operator
-	// +kubebuilder:validation:Optional
-	Version string `json:"version,omitempty"`
-
-	// ReleaseImage is the release image applied to the hosted control plane.
-	ReleaseImage string `json:"releaseImage,omitempty"`
```

Comment:
FIXME: We'd need to keep this field around and deprecate it to avoid breaking deserializing-from-etcd in existing clusters updating to this new HyperShift operator version.

Reply:
Agree, this needs to be a compatible change

Reply:
I've restored this as a deprecated property in d2909321.